gui/src/2D/controls/textBlock.ts

```diff
@@ -381,6 +468,124 @@ export class TextBlock extends Control {
         }
     }
 
+    //++CR
+    private _drawStructuredText(structuredText: StructuredText, textWidth: number, y: number, context: ICanvasRenderingContext): void {
+        var attr ,
+            x = 0 ,
+            width = this._currentMeasure.width ;

```

Comment:
Pattern using commas repeats throughout the code.
Variables should be declared each on their own line with let & const

>let x = 0;
const width = this._currentMeasure.width;

attr should be declared where it's used, e.g. line 498